lib/octicons_react/src/index.d.ts

```diff
@@ -1,14 +1,19 @@
+// eslint-disable-next-line import/no-namespace
 import * as React from 'react'
 
-import {Icon} from './__generated__/icons'
+import { Icon } from './__generated__/icons'
 
 type Size = 'small' | 'medium' | 'large'
 
 export interface OcticonProps {
   'aria-label'?: string
+  'aria-labelby'?: string
   tabIndex?: number
   children?: React.ReactElement<any>
   className?: string
+  title?: string | React.ReactElement<any>
```

Comment:
Do you have a preference between `title` as a prop used for `<title>` or accepting `children` as a prop so people can supply it manually? e.g.

```tsx
<SomeIcon title="Some icon" />

// Versus
<SomeIcon>
  <title>Some icon</title>
</SomeIcon>
```

I assume people are okay with the former version but just wanted to ask.

Reply:
Hey, I think since this is for icons, that title would be just a prop for accessibility reasons.